src/core/agent/agent.ts

```diff
@@ -399,6 +399,21 @@ class Agent {
     return { bran, mnemonic: entropyToMnemonic(passcodeBytes) };
   }
 
+  async getMnemonic(): Promise<string> {
+    return this.convertToMnemonic(await this.getBran());
+  }
+
+  private convertToMnemonic(bran: string): string {
+    // This converts the 21 character Signify-TS passcode/bran to a BIP-39 compatible word list.
+    // The passcode is assumed as UTF-8 in our recovery. In actuality, it is the qb64 CESR salt without the code.
+    // We believe it's easier to encode it as UTF-8 in case there is a change in Signify TS in how the passcode is handled.
+    const passcodeBytes = Buffer.concat([
+      Buffer.from(bran, "utf-8"),
+      Buffer.alloc(3),
+    ]);
+    return entropyToMnemonic(passcodeBytes);
```

Comment:
@Sotatek-DukeVu The code I sent you also updated `getBranAndMnemonic`. I want it to reuse `convertToMnemonic` - otherwise it's implemented twice with the same comments.

Reply:
Done

Reply:
thanks!